.github/workflows/test-package-build.yml

```diff
@@ -28,23 +32,36 @@ jobs:
           npm install
           npm run build
 
-      - name: Build wheel
+      - name: Install dependencies
+        run: |
+          pip install wheel twine
+
+      - name: Build distribution files
         id: build-wheel
         run: |
-          pip install wheel
-          python setup.py bdist_wheel
+          if [ "${{ matrix.type }}" = "skinny" ]; then
+            export MLFLOW_SKINNY=true
+          fi
+
+          python setup.py sdist bdist_wheel
+          ls -lh dist
```

Comment:
Do we need this line?

Reply:
This line outputs:

```
-rw-r--r-- 1 runner docker 500K Feb  5 09:26 mlflow-1.13.2.dev0-py3-none-any.whl
-rw-r--r-- 1 runner docker 390K Feb  5 09:26 mlflow-1.13.2.dev0.tar.gz
```

I added this line to confirm dist files are successfully generated + check their file sizes.